src/modules/auth/dto/signIn.dto.ts

```diff
@@ -10,4 +10,8 @@ export default class SignInDto {
   @IsOptional()
   @IsString()
   readonly password?: string;
+
+  @IsBoolean()
+  @IsOptional()
+  readonly is_google?: boolean;
```

Comment:
why you need this came from BE? you should already know this information if user sign in from google

Reply:
fixed